### DISCUSSION TITLE
Oracle - create timestamp with local timezone

### DISCUSSION THREAD

**Discussion by mtuxgu** [sqlalchemy/sqlalchemy#9085] [Usage Questions] [answered]
Dear community,
does anyone ever tried to create a field in Oracle with datatype TIMESTAMP WITH LOCAL TIMEZONE (https://docs.oracle.com/cd/B14117_01/server.101/b10749/ch4datetime.htm#i1006169) ?

When I try to create such a field with an alembic migration using the following code:
```py
import sqlalchemy as sa
 from sqlalchemy import Identity 
from sqlalchemy.dialects.oracle.cx_oracle 
import oracle from alembic import op 
(...)
sa.Column(     
  "AUDIT_TIMESTAMP",     
  oracle.TIMESTAMP(timezone=True),     
  server_default=sa.text("current_timestamp"),     
  nullable=False,     
  comment="Timestamp of audit-entry",
 ), 
```
I always get the field created as TIMESTAMP WITH TIMEZONE (what is significant different...).

I am not sure if it is actually possible to do this with alembic?

BR
Markus

**Comment by CaselIT** [comment, marked as answer]
Hi,

No, sqlalchemy does not seem to support it.
Moving to sqlalchemy since the issue is in sqlalchemy, not alembic

**Reply by mtuxgu** [reply]
Hi, 

thx for clarification - i will create an issue for that topic...

**Comment by mtuxgu** [comment]
Create an issue from this discussion: https://github.com/sqlalchemy/sqlalchemy/issues/9086
